### ISSUE TITLE
Can not use <C-N><C-P> move down/up select match world in popup-menu

### ISSUE THREAD

**Issue by ihexon** [wsdjeg/SpaceVim#1558]
<!-- bug reporting without issue template will be closed automatically -->

I want use Ctrn-N CTRL-P to to scroll through the popup menu, but When I `<C-N>`, It select first one and auto insert, and the Completion popup-menu immediately changes to matches the first one word,  so  `<C-N><C-P>` can not move down to select second word.....

I What select the `bin` dir 
![screenshot_2018-03-20_08-58-57](https://user-images.githubusercontent.com/14349453/37939932-bc3a679e-3133-11e8-87d0-d870c8f71231.png)

Just like this
![screenshot_2018-03-22_01-06-52](https://user-images.githubusercontent.com/14349453/37940018-5ab376b8-3134-11e8-8dd9-df2cec031e25.png)

BUT I `<C-N>`, the pop-menu changes to match only first one
![screenshot_2018-03-22_01-06-52](https://user-images.githubusercontent.com/14349453/37939985-1523d002-3134-11e8-8f0e-05e06bc38d33.png)


in Neovim it can be scroll through the popup menu, but vim have problem....


## The reproduce ways from Vim starting (Required!)
Terminal,GVIM

## Debug info
```viml                                                                                                                                                                                                              
g:spacevim_statusline_left_sections = ['winnr', 'filename', 'major mode', 'syntax checking', 'minor mode lighters', 'vcs', 'hunks']                                                                                  
g:spacevim_enable_statusline_display_mode = 0                                                                                                                                                                        
g:spacevim_enable_os_fileformat_icon = 1                                                                                                                                                                             
g:spacevim_leader_guide_position = 'botright'                                                                                                                                                                        
g:spacevim_version = '0.8.0-dev'                                                                                                                                                                                     
g:spacevim_force_global_config = 1                                                                                                                                                                                   
g:spacevim_smartcloseignoreft = ['help', 'tagbar', 'vimfiler', 'SpaceVimRunner', 'SpaceVimREPL', 'SpaceVimQuickFix', 'HelpDescribe', 'VebuggerShell', 'VebuggerTerminal']                                            
g:spacevim_default_indent = 2                                                                                                                                                                                        
g:spacevim_enable_javacomplete2_py = 0                                                                                                                                                                               
g:spacevim_enable_powerline_fonts = 1                                                                                                                                                                                
g:spacevim_statusline_separator = 'arrow'                                                                                                                                                                            
g:spacevim_enable_language_specific_leader = 1                                                                                                                                                                       
g:spacevim_warning_symbol = '⚠'                                                                                                                                                                                      
g:spacevim_simple_mode = 0                                                                                                                                                                                           
g:spacevim_colorscheme = 'gruvbox'                                                                                                                                                                                   
g:spacevim_enable_guicolors = 1                                                                                                                                                                                      
g:spacevim_custom_plugins = [['mattn/webapi-vim'], ['vimwiki/vimwiki'], ['itchyny/calendar.vim']]                                                                                                                    
g:spacevim_leader_guide_submode_mappings = {'<C-C>': 'win_close'}                                                                                                                                                    
g:spacevim_buffer_index_type = 1                                                                                                                                                                                     
g:spacevim_hiddenfileinfo = 1                                                                                                                                                                                        
g:spacevim_statusline_inactive_separator = 'arrow'                                                                                                                                                                   
g:spacevim_gitcommit_pr_icon = ''                                                                                                                                                                                   
g:spacevim_plugin_manager = 'dein'                                                                                                                                                                                   
g:spacevim_error_symbol = '✖'                                                                                                                                                                                        
g:spacevim_vim_plug_installed = 0                                                                                                                                                                                    
g:spacevim_windows_smartclose = 'q'                                                                                                                                                                                  
g:spacevim_relativenumber = 1                                                                                                                                                                                        
g:spacevim_windows_leader = 's'                                                                                                                                                                                      
g:spacevim_wildignore = '*/tmp/*,*.so,*.swp,*.zip,*.class,tags,*.jpg,*.ttf,*.TTF,*.png,*/target/*,.git,.svn,.hg,.DS_Store,*.svg'                                                                                     
g:spacevim_enable_ale = 0                                                                                                                                                                                            
g:spacevim_enable_insert_leader = 1                                                                                                                                                                                  
g:spacevim_search_tools = ['rg', 'ag', 'pt', 'ack', 'grep']                                                                                                                                                          
g:spacevim_colorscheme_bg = 'dark'                                                                                                                                                                                   
g:spacevim_enable_neocomplcache = 0                                                                                                                                                                                  
g:spacevim_leader_guide_sort_horizontal = 0                                                                                                                                                                          
g:spacevim_enable_vimfiler_welcome = 1                                                                                                                                                                               
g:spacevim_max_column = 120                                                                                                                                                                                          
g:spacevim_enable_debug = 1                                                                                                                                                                                          
g:spacevim_enable_key_frequency = 0                                                                                                                                                                                  
g:spacevim_auto_disable_touchpad = 1                                                                                                                                                                                 
g:spacevim_vim_help_language = 'en'                                                                                                                                                                                  
g:spacevim_gitcommit_issue_icon = ''                                                                                                                                                                                
g:spacevim_lint_on_save = 1                                                                                                                                                                                          
g:spacevim_layer_lang_java_formatter = '/home/zhuzhihao/.SpaceVim/bin/google-java-format-1.5-all-deps.jar'                                                                                                           
g:spacevim_dein_installed = 1                                                                                                                                                                                        
g:spacevim_autocomplete_method = 'deoplete'                                                                                                                                                                          
g:spacevim_enable_vimfiler_filetypeicon = 0                                                                                                                                                                          
g:spacevim_plugin_manager_max_processes = 16                                                                                                                                                                         
g:spacevim_statusline_unicode_symbols = 1                                                                                                                                                                            
g:spacevim_statusline_right_sections = ['fileformat', 'cursorpos', 'percentage']                                                                                                                                     
g:spacevim_unite_leader = '\f'                                                                                                                                                                                       
g:spacevim_enable_ycm = 0                                                                                                                                                                                            
g:spacevim_enable_tabline_filetype_icon = 1                                                                                                                                                                          
g:spacevim_colorscheme_default = 'desert'                                                                                                                                                                            
g:spacevim_project_rooter_automatically = 1                                                                                                                                                                          
g:spacevim_neobundle_installed = 0                                                                                                                                                                                   
g:spacevim_disabled_plugins = []         
g:spacevim_leader_guide_max_size = 0                                                                                                                                                                                 
g:spacevim_guifont = ''                                                                                                                                                                                              
g:spacevim_info_symbol = 'ⓘ'                                                                                                                                                                                         
g:spacevim_leader_guide_flatten = 1                                                                                                                                                                                  
g:spacevim_sidebar_width = 30                                                                                                                                                                                        
g:spacevim_enable_googlesuggest = 0                                                                                                                                                                                  
g:spacevim_enable_neomake = 1                                                                                                                                                                                        
g:spacevim_plugin_bundle_dir = '/home/zhuzhihao/.cache/vimfiles/'                                                                                                                                                    
g:spacevim_leader_guide_run_map_on_popup = 1                                                                                                                                                                         
g:spacevim_hosts_url = 'https://raw.githubusercontent.com/racaljk/hosts/master/hosts'                                                                                                                                
g:spacevim_leader_guide_default_group_name = ''                                                                                                                                                                      
g:spacevim_smartcloseignorewin = ['__Tagbar__', 'vimfiler:default']                                                                                                                                                  
g:spacevim_terminal_cursor_shape = 2                                                                                                                                                                                 
g:spacevim_altmoveignoreft = ['Tagbar', 'vimfiler']                                                                                                                                                                  
g:spacevim_keep_server_alive = 1                                                                                                                                                                                     
g:spacevim_snippet_engine = 'neosnippet'                                                                                                                                                                             
g:spacevim_leader_guide_vertical = 0                                                                                                                                                                                 
g:spacevim_project_rooter_patterns = ['.git/', '_darcs/', '.hg/', '.bzr/', '.svn/', '.clang', 'package.json', 'pom.xml']                                                                                             
g:spacevim_enable_cursorcolumn = 0                                                                                                                                                                                   
g:spacevim_realtime_leader_guide = 1                                                                                                                                                                                 
g:spacevim_github_username = ''                                                                                                                                                                                      
g:spacevim_leader_guide_hspace = 5                                                                                                                                                                                   
g:spacevim_denite_leader = 'F'                                                                                                                                                                                       
g:spacevim_checkinstall = 1                                                                                                                                                                                          
g:spacevim_windows_index_type = 0                                                                                                                                                                                    
g:spacevim_filetype_icons = {}                                                                                                                                                                                       
g:spacevim_plugin_name = 'vim-fugitive'                                                                                                                                                                              
g:spacevim_enable_cursorline = 1                                                                                                                                                                                     
g:spacevim_custom_color_palette = []                                                                                                                                                                                 
g:spacevim_lint_on_the_fly = 0                                                                                                                                                                                       
g:spacevim_debug_level = 1                                                                                                                                                                                           
g:spacevim_filemanager = 'vimfiler'                                                                                                                                                                                  
g:spacevim_src_root = 'E:\sources\'                                                                                                                                                                                  
g:spacevim_language = ''                                                                                                                                                                                             
g:spacevim_enable_vimfiler_gitstatus = 0     
### SpaceVim Health checking :                                                                                                                                                                                       
                                                                                                                                                                                                                     
SpaceVim clipboard support check report:                                                                                                                                                                             
Checking +clipboard:                                                                                                                                                                                                 
      SUCCEED!                                                                                                                                                                                                       
SpaceVim environment check report:                                                                                                                                                                                   
Current progpath: vim(/home/zhuzhihao/vim_bins/bin/vim)                                                                                                                                                              
version: 800                                                                                                                                                                                                         
OS: Linux                                                                                                                                                                                                            
[shell, shellcmdflag, shellslash]: ['/usr/bin/zsh', '-c', 0]                                                                                                                                                         
SpaceVim lua support check report:                                                                                                                                                                                   
Checking +lua:                                                                                                                                                                                                       
      SUCCEED!                                                                                                                                                                                                       
SpaceVim python support check report:                                                                                                                                                                                
Checking +python3:                                                                                                                                                                                                   
      SUCCEED!                                                                                                                                                                                                       
Checking +python:                                                                                                                                                                                                    
      SUCCEED!                                                                                                                                                                                                       
                                                                                                                                                                                                                     
### SpaceVim runtime log :                                                                                                                                                                                           
                                                                                                                                                                                                                     
```log                                                                                                                                                                                                               
[ SpaceVim ] : logger file  does not exists, only log for current process will be shown!                                                                                                                             
[ SpaceVim ] [20:16:50] [ Info ] SpaceVim server startup at:/tmp/spacevim_vim_server                                                                                                                                 
[ SpaceVim ] [20:16:50] [ Info ] Find project root('.git/',''):/home/zhuzhihao/.SpaceVim/.git/                                                                                                                       
[ SpaceVim ] [20:16:50] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
[ SpaceVim ] [20:16:50] [ Info ] Find project root('.git/','/home/zhuzhihao/.SpaceVim/Startify'):/home/zhuzhihao/.SpaceVim/.git/                                                                                     
[ SpaceVim ] [20:16:50] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
[ SpaceVim ] [20:16:53] [ Info ] Find project root('.git/','/home/zhuzhihao/.SpaceVim/[denite]'):/home/zhuzhihao/.SpaceVim/.git/                                                                                     
[ SpaceVim ] [20:16:53] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
[ SpaceVim ] [20:17:09] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
[ SpaceVim ] [20:17:09] [ Info ] Find project root('.git/',''):/home/zhuzhihao/.SpaceVim/.git/                                                                                                                       
[ SpaceVim ] [20:17:09] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
[ SpaceVim ] [20:17:09] [ Info ] Find project root('.git/','/home/zhuzhihao/.SpaceVim/doc/SpaceVim.txt'):/home/zhuzhihao/.SpaceVim/.git/                                                                             
[ SpaceVim ] [20:17:09] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
[ SpaceVim ] [20:17:24] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
[ SpaceVim ] [20:17:54] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
[ SpaceVim ] [20:18:35] [ Info ] change to root:/home/zhuzhihao/.SpaceVim                                                                                                                                            
```

**Comment by Shougo** [comment]
I get it.
It seems deoplete problem.

Please wait.

**Comment by wsdjeg** [comment]
@ihexon @Shougo  I can not reproduce it in neovim, does it has been fixed?

https://asciinema.org/a/UNeKRtH6pa6RQtg2taiGd1SiN

**Comment by Shougo** [comment]
Yes.

**Comment by Shougo** [comment]
It is fixed in #704.

**Comment by wsdjeg** [comment]
Thanks.
